app/Domain/Integrations/Repositories/EloquentIntegrationRepository.php

```diff
@@ -115,15 +129,25 @@ public function activate(UuidInterface $id): void
         $integrationModel->activate();
     }
 
-    public function activateWithOrganization(UuidInterface $id, UuidInterface $organizationId, ?string $couponCode): void
+    public function activateWithOrganization(UuidInterface $integrationId, UuidInterface $organizationId, ?string $couponCode, array $organizers=[]): void
```

Comment:
Should this also be typed on the new collection?

Reply:
I thought so to at first but if you look at ActivateIntegration you can see that actually it is an array of strings (ids).
I will add a docblock to make this more clear.

Reply:
But should it be an array of strings? The `UdbOrganizers` could be created where this gets called. So in the `handle` method of `ActivateUitpasIntegration` the construction of `UdbOrganizer` could be done.

Reply:
Moved it to the ActivateUitpasIntegration class